<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFamilyDialog</class>
 <widget class="QDialog" name="EditFamilyDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Family</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayoutMain">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFamilyName">
       <property name="text">
        <string>Name*:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBase">
       <property name="text">
        <string>Base Family:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayoutBase">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonBaseYes">
            <property name="toolTip">
             <string>Select this to make the family a base family.
A base family is displayed at the top of family tree (subject to sorting amongst the base families).
It is okay if you have no base family or vice versa.</string>
            </property>
            <property name="statusTip">
             <string>Base Family: Yes</string>
            </property>
            <property name="whatsThis">
             <string>Select this to make the family a base family.
A base family is displayed at the top of family tree (subject to sorting amongst the base families).
It is okay if you have no base family or vice versa.</string>
            </property>
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBaseNo">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Select this to make the family a normal family.
A normal family is displayed after the base families in a sorted order.</string>
            </property>
            <property name="statusTip">
             <string>Base Family: No</string>
            </property>
            <property name="whatsThis">
             <string>Select this to make the family a normal family.
A normal family is displayed after the base families in a sorted order.</string>
            </property>
            <property name="text">
             <string>No</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAlisases">
       <property name="text">
        <string>Aliases:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayoutAliases">
       <item>
        <widget class="QListView" name="listViewAliases">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutAliasesButtons">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonAdd">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonDel">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonUp">
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonDown">
              <property name="text">
               <string>Down</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelMembers">
       <property name="text">
        <string>Members:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="labelFamilyMembers">
       <property name="toolTip">
        <string>Primary members of this family. Use Tree View to edit list of primary Members.</string>
       </property>
       <property name="statusTip">
        <string>Primary members</string>
       </property>
       <property name="whatsThis">
        <string>Primary members of this family. Use Tree View to edit list of primary Members.</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>Space for primary members.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelCreatedOn">
       <property name="text">
        <string>Created On:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelModifiedOn">
       <property name="text">
        <string>Last Modified:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelModificationCount">
       <property name="text">
        <string>Modification Count:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="toolTip">
        <string>Click ok to accept the changes.
Click cancle to discard the changes.</string>
       </property>
       <property name="whatsThis">
        <string>Click ok to accept the changes.
Click cancle to discard the changes.</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumberModificationCount">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelModifiedOnValue">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>Wed, 27 Oct 2010 22:19:25</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelCreatedOnValue">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>Wed, 27 Oct 2010 22:19:25</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditName">
          <property name="toolTip">
           <string>Name of the family goes here.
 It can consist of any text (Its after all your tree).</string>
          </property>
          <property name="statusTip">
           <string>Family Name</string>
          </property>
          <property name="whatsThis">
           <string>Name of the family goes here. It can consist of any text (Its after all your tree).</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MainWindow/MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditFamilyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditFamilyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
